### PR TITLE
Add Conveyor packaging.

### PR DESCRIPTION
**Pull request by mikehearn**
Hey there,

AtlantaFX is a really nice piece of work! I figured more people would be interested if it were easier to run the sampler and follow the libraries evolution, so packaged it with [Conveyor](https://conveyor.hydraulic.dev). This PR implements the packaging and you can see the download site here:

https://downloads.hydraulic.dev/atlantafx/sampler/download.html

I've also signed the downloads so there aren't any security warnings. I hope this contribution is welcome and that AtlantaFX can become a new de-facto standard skin for JavaFX apps. When you want to do new releases, just open an issue in this fork and we'll rebuild/push an update.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ JavaFX CSS theme collection plus additional controls. It's inspired by [FlatLaf]
 ![primer_light](./.screenshots/primer_light.png)
 ![primer_dark](./.screenshots/primer_dark.png)
 
+## Try it out
+
+Grab a **[self-updating download of the Sampler app](https://downloads.hydraulic.dev/atlantafx/sampler)** for Windows, macOS and Linux, packaged with [Conveyor](https://www.hydraulic.software).
```
Comment (tedyoung) [inline README.md:12]: Shouldn't this link be

https://downloads.hydraulic.dev/atlantafx/sampler/download.html

the link without the /download.html currently 404s.

Comment (mikehearn) [reply README.md]: Oof, yes, you're right. I just pushed a fixed commit - thanks!

**Comment by tedyoung** [comment]
Looks like the resulting macOS app requires 11.6 or later (currently stuck on 10.15.7), but I don't see minimum macOS requirements anywhere?

**Comment by mikehearn** [comment]
Ah, good find, thanks for testing! How come you got stuck on such an old version?

We have a version skew in the defaults. The package declares that `LSMinimumSystemVersion = 10.11.0`, so pretty old, but the launcher binary itself seems to have picked up some Xcode default and declares in the binary headers that it needs 11.6 - that's not an explicit choice we made, so it must be some Apple policy. The binary is very simple so it should be easy to recompile it to target older macOS versions.

**Comment by mikehearn** [comment]
@tedyoung If you don't mind, could you re-download the Mac package and see if it works now? We've set the min version to 10.14 and made it consistent.

**Comment by tedyoung** [comment]
@mikehearn I was able to open the app, but it "quit unexpectedly". (I don't want to pollute this PR with the crash info, so let me know where to send it.)

> stuck on such an old version

There's a couple of apps that I use that took a loooong time to get updated to run on 11.x and later macOS. Now it's just me finding the time to do the upgrade. 😬

**Comment by tedyoung** [comment]
I'll paste a few tidbits, maybe it's enough?

```
Notes:                 Translocated Process

Crashed Thread:        0

Exception Type:        EXC_CRASH (Code Signature Invalid)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    Namespace CODESIGNING, Code 0x1
```

**Comment by mkpaz** [comment]
Thanks! Sure, I'll gladly accept any PR that makes the project better.

**Comment by mikehearn** [comment]
Thanks @mkpaz - just to double check, does it work for you?

@tedyoung Hmm the Mac app works fine here, so this is probably another old OS issue. Unfortunately it's remarkably hard to install old versions of macOS into Parallels, as Apple make it hard to actually obtain the installer, so testing this is tricky. If you could mail a full crash log to contact@hydraulic.software that'd be much appreciated, if you could also include the output of `codesign -v -vvvv AtlantaFX\ Sampler.app` then that'd be even better. Perhaps there's a clue. Otherwise we'll try and locate old macOS ISOs/installers so we can repro this directly.

**Comment by mkpaz** [comment]
> Thanks @mkpaz - just to double check, does it work for you?

Yes, though I've only `tar.gz` version.

**Comment by mikehearn** [comment]
OK - if you don't mind me asking, which distro are you on?

**Comment by mkpaz** [comment]
Fedora 36, vanilla Gnome 3.

**Comment by mikehearn** [comment]
The issue with macOS Catalina has been resolved.

**Comment by ennerf** [comment]
@mikehearn 1.1.0 is out

**Comment by mikehearn** [comment]
Thanks! Will get the sampler updating within a day or so.

**Comment by ennerf** [comment]
@mikehearn v1.2.0

**Comment by mikehearn** [comment]
Thanks, was on holiday. It's released now!

**Comment by mikehearn** [comment]
It's done already.

**Pull request merged**